<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="wgtBack" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="wgtTitle" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#wgtTitle
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.25
 rgb(3,154,184, 240), stop:1 rgb(3,154,184,240));
border: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.25
 rgb(146, 217, 255), stop:1 rgb(255,255,255,150));;
border-radius: 1px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbIcon">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/MainWindow/res/cmd.png);
margin-left: 5px;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbTitle">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Cascadia Code&quot;;
color: rgb(255, 255, 255);
margin-left: 5px;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;remote-cmd&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="close_btn">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 10pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 00);
border: 0px solid #ffffff;
border-radius: 2px;
margin-left: 5px;
margin-right: 5px;
}
QPushButton:hover
{
font: 10pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 32);
}
QPushButton:pressed
{
font: 10pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 16);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/MainWindow/res/close.png</normaloff>:/MainWindow/res/close.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="wgtMain" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#wgtMain
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.25
 rgb(3,65,88, 240), stop:1 rgb(3,65,88, 240));
border: 0px solid #ffffff;
border-radius: 1px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="styleSheet">
             <string notr="true">QTabWidget::pane 
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.25
 rgb(3,65,88, 240), stop:1 rgb(3,65,88, 240));
border: 0px solid #ffffff;
border-radius: 1px;
}</string>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
  <include location="../../../cpp-old/Titan/DataAnly/DataAnly.qrc"/>
 </resources>
 <connections/>
</ui>
